### ISSUE TITLE
Command-line argument for output file

### ISSUE THREAD

**Issue by ept** [ept/avrodoc#4]
For situations where you can't redirect stdout (e.g. some CI servers), we should support writing the HTML output to a file specified on the command line.

I think [sort(1)](http://linux.die.net/man/1/sort) is a good example for command line style. In its style, we would support all these forms (argument order doesn't matter, so the output argument can come before or after the inputs):

```
$ avrodoc input1.avsc input2.avsc > output.html
$ avrodoc input1.avsc input2.avsc -o output.html
$ avrodoc input1.avsc input2.avsc --output output.html
$ avrodoc input1.avsc input2.avsc --output=output.html
$ avrodoc -o output.html -- -input-file-that-starts-with-a-hyphen.avsc
```

**Comment by ept** [comment]
See [this thread](https://groups.google.com/forum/?fromgroups=#!topic/nodejs/TM_fUahX3TI) on node.js command-line argument parsers.

**Comment by mphuff** [comment]
This seems like by far the most flexible offering and is exactly I think what we're after: https://github.com/substack/node-optimist

I'm hoping that I'll get to this this week but likely at latest early next week. That is unless you beat me to it.

Micah

**Comment by mphuff** [comment]
Glad to see this got pulled in so quickly! Let me know when you get a chance to get this out to the npmjs site.

Thank you for all of your help Martin! 

Micah

**Comment by ept** [comment]
Fixed in #8, thanks Micah :)
